RxExample/RxExample/Examples/GitHubSearchRepositories/GitHubSearchRepositoriesViewController.swift

```diff
@@ -43,8 +43,9 @@ class GitHubSearchRepositoriesViewController: ViewController, UITableViewDelegat
 
 
         let loadNextPageTrigger = self.tableView.rx.contentOffset
-            .flatMap { _ in
-                self.tableView.isNearBottomEdge(edgeOffset: 20.0)
+            .flatMap { [weak self] _ -> Observable<Void> in
+                guard let `self` = self else { return Observable.empty() }
+                return self.tableView.isNearBottomEdge(edgeOffset: 20.0)
```

Comment:
... so we can just say here `return tableView.isNearBottomEdge(edgeOffset: 20.0)` without self.